ghost/api-framework/lib/pipeline.js

```diff
@@ -230,6 +230,9 @@ const pipeline = (apiController, apiUtils, apiType) => {
             frame.method = method;
 
             let cacheKey = JSON.stringify(frame.options);
```

Comment:
We can save some time doing this serialization only when needed. `JSON.stringify` is not a crazy expensive operation but who knows what's in the frame options and would add up unnecessarily 

What I mean is:
```
let cacheKey;

if (apiImpl.generateCacheKey)
   cacheKey = await apiImpl.generateCacheKey(frame);
} else {
  // default key
  cacheKey = JSON.stringify(frame.options);
}

Reply:
Yup! And can also use the stable-json-stringify here too

Reply:
Agreed!